docs/tab-widgets/install-agents.asciidoc

```diff
@@ -0,0 +1,433 @@
+from elasticapm.contrib.flask import ElasticAPM
+app = Flask(__name__)
+apm = ElasticAPM(app)
+
+# or configure to use ELASTIC_APM in your application's settings
+from elasticapm.contrib.flask import ElasticAPM
+app.config['ELASTIC_APM'] = {
+  # Set required service name. Allowed characters:
+  # a-z, A-Z, 0-9, -, _, and space
+  'SERVICE_NAME': '',
+
+  # Use if APM Server requires a token
+  'SECRET_TOKEN': '',
+
+  # Set custom APM Server URL (default: http://localhost:8200)
+  'SERVER_URL': '',
+}
+
+apm = ElasticAPM(app)
+----
+
+*Learn more in the agent reference*
+
+* {apm-py-ref-v}/supported-technologies.html[Supported technologies]
+* {apm-py-ref-v}/configuration.html[Advanced configuration]
+
+
+// end::python[]
+
+// tag::ruby[]
+*Install the APM agent*
+
+Add the agent to your Gemfile.
+
+[source,ruby]
+----
+gem 'elastic-apm'
+----
++
+*Configure the Agent*
+
+Ruby on Rails::
++
+APM is automatically started when your app boots.
+Configure the agent by creating the config file `config/elastic_apm.yml`:
++
+[source,ruby]
+----
+# config/elastic_apm.yml:
+
+# Set service name - allowed characters: a-z, A-Z, 0-9, -, _ and space
+# Defaults to the name of your Rails app
+service_name: 'my-service'
+
+# Use if APM Server requires a token
+secret_token: ''
+
+# Set custom APM Server URL (default: http://localhost:8200)
+server_url: 'http://localhost:8200'
+----
+
+Rack::
++
+For Rack or a compatible framework (e.g. Sinatra), include the middleware in your app and start the agent.
++
+[source,ruby]
+----
+# config.ru
+  require 'sinatra/base'
+
+  class MySinatraApp < Sinatra::Base
+    use ElasticAPM::Middleware
+
+    # ...
+  end
+
+  ElasticAPM.start(
+    app: MySinatraApp, # required
+    config_file: '' # optional, defaults to config/elastic_apm.yml
+  )
+
+  run MySinatraApp
+
+  at_exit { ElasticAPM.stop }
+----
++
+*Create a config file*
++
+Create a config file config/elastic_apm.yml:
++
+[source,ruby]
+----
+# config/elastic_apm.yml:
+
+# Set service name - allowed characters: a-z, A-Z, 0-9, -, _ and space
+# Defaults to the name of your Rack app's class.
+service_name: 'my-service'
+
+# Use if APM Server requires a token
+secret_token: ''
+
+# Set custom APM Server URL (default: http://localhost:8200)
+server_url: 'http://localhost:8200'
+----
+
+*Learn more in the agent reference*
+
+* {apm-ruby-ref-v}/supported-technologies.html[Supported technologies]
+* {apm-ruby-ref-v}/configuration.html[Advanced configuration]
+
+// end::ruby[]
+
+// ***************************************************
+// ***************************************************
+
+// tag::rum[]
+*Enable Real User Monitoring support in APM Server*
+
+APM Server disables RUM support by default.
+To enable it, set `apm-server.rum.enabled: true` in your APM Server configuration file.
+
+*Set up the Agent as a dependency*
+
+You can install the Agent as a dependency to your application with `npm install @elastic/apm-rum --save`.
```

Comment:
I've noticed a little inconsistency across these tabs wrt how you refer to the agent. Sometimes you say APM agent, sometimes agent (lowercase), and sometimes Agent uppercase. It's kind of minor but might be worth fixing in this PR.

Reply:
👍